tests/otel_integrations/test_openai.py

```diff
@@ -277,24 +277,7 @@ def test_sync_chat_completions(instrumented_client: openai.Client, exporter: Tes
                     'logfire.msg': "Chat Completion with 'gpt-4'",
                     'logfire.span_type': 'span',
                     'logfire.tags': ('LLM',),
-                    'response_data': IsJson(
-                        {
-                            'message': {
-                                'content': 'Nine',
-                                'refusal': None,
-                                'role': 'assistant',
-                                'function_call': None,
-                                'tool_calls': None,
-                            },
-                            'usage': {
-                                'completion_tokens': 1,
-                                'prompt_tokens': 2,
-                                'total_tokens': 3,
-                                'completion_tokens_details': None,
-                                'prompt_tokens_details': None,
-                            },
-                        }
-                    ),
+                    'response_data': '{"message":{"content":"Nine","refusal":null,"role":"assistant","audio":null,"function_call":null,"tool_calls":null},"usage":{"completion_tokens":1,"prompt_tokens":2,"total_tokens":3,"completion_tokens_details":null,"prompt_tokens_details":null}}',
```

Comment:
The `IsJson` was cool here... 👀

Reply:
Can you bring it back? I think only the `audio` field was added?

Reply:
Yeah, will try